examples/init/INCAR_methane.md

```diff
@@ -1,21 +1,33 @@
 PREC=A
-ENCUT=400
+ENCUT=400.000000
 ISYM=0
-ALGO=Fast
-EDIFF=1.000000e-06
-LREAL=False
+ALGO=fast
+EDIFF=1E-6
+LREAL=F
 NPAR=4
 KPAR=1
-NELM=120
-NELMIN=4
+
+NELM=200
+ISTART=0
+ICHARG=2
 ISIF=2
-ISMEAR=0
-SIGMA=0.20000
+ISMEAR=1
```

Comment:
Why changing ISMEAR to 1?

Reply:
Thanks, you're right. should be IMSEAR = 0 for insulators like methane, fixed